docs/LINUX.md

```diff
@@ -1,125 +1,141 @@
 # LogDNA Agent on Linux distributions
 
-The agent supports 64-bit (`x86_64`) Linux distributions and provides rpm and deb packages.
+The LogDNA Agent for Linux collects log data from your Linux environment. The agent is available as a Linux package for Debian-based distributions using `apt` and Red Hat-based distributions using `yum`. If you use a different package manager that does not support `.deb` or `.rpm` files, you can still use the agent by [manually compiling the binary](https://github.com/logdna/logdna-agent-v2/blob/eb06d4f3f7c1033b494f1f0439957f96533f9225/docs/README.md#building-agent-binary-on-linux).
 
-## Installing
+## Table of Contents
+  * [Installation: fresh installs](#Installation)
+  * [Upgrading: migrating from legacy Linux agent](#upgrade-migration)
+  * [Usage](#usage)
+    * [Enable the agent](#enable-the-agent)
+    * [Configure the agent](#configure-the-agent)
+    * [Run the agent](#run-the-agent)
 
-### On Debian-based distributions
+## Installation
 
-```shell script
+1. To download the agent and resources package, open a host terminal and run the appropriate commands based on the Linux distribution.
+
+* Debian-based distributions
```

Comment:
style-wise, with the bullets, it's harder to read `Debian-based distributions` and `RPM/Red Hat-based distributions`. Can we bold it instead?